### ISSUE TITLE
Admin portal crash: can't locate momentjs-timezone

### ISSUE THREAD

**Issue by nebulade** [TryGhost/Ghost#17932]
### Issue Summary

While updating the Ghost package for Cloudron, the installation crashes when the admin portal is opened in the browser:
```
Uncaught Error: Could not find module `moment-timezone` imported from `ghost-admin/app`
```
With the same [Dockerimage](https://git.cloudron.io/cloudron/ghost-app/-/blob/master/Dockerfile?ref_type=heads#L20) version 5.60.0 produces a working installation.

Thanks for any pointers on how to debug this.

### Steps to Reproduce

Installed with:
```
ghost install 5.61.0 --no-prompt --no-stack --no-setup --no-start --dbhost notlocalhost --dir /home/cloudron/ghost
```

### Ghost Version

5.61.0

### Node.js Version

v18.12.1

### How did you install Ghost?

Cloudron / Docker

### Database type

MySQL 8

### Code of Conduct

- [X] I agree to be friendly and polite to people in this repository

**Comment by elguenny** [comment]
I can confirm the error with a normal installation

**Comment by khalid-faiz** [comment]
Yeb, I tried that with a normal VPS, and also replicated the issue offline. This is a whole console log when trying to attempt to go localhost/ghost.
```
GET
http://localhost:2368/ghost/assets/chunk.208.dbf172ad32f72f21a5dc.js

The resource from “http://localhost:2368/ghost/assets/chunk.208.dbf172ad32f72f21a5dc.js” was blocked due to MIME type (“text/html”) mismatch (X-Content-Type-Options: nosniff).
[ghost](http://localhost:2368/ghost/)
InstallTrigger is deprecated and will be removed in the future. [vendor-3631184082d609038638c1e169a002e7.js:550:24](http://localhost:2368/ghost/assets/vendor-3631184082d609038638c1e169a002e7.js)
GET
http://localhost:2368/ghost/assets/chunk.208.dbf172ad32f72f21a5dc.js

Loading failed for the <script> with source “http://localhost:2368/ghost/assets/chunk.208.dbf172ad32f72f21a5dc.js”. [ghost:60:55](http://localhost:2368/ghost/)
The resource from “http://localhost:2368/ghost/assets/chunk.208.dbf172ad32f72f21a5dc.js” was blocked due to MIME type (“text/html”) mismatch (X-Content-Type-Options: nosniff).
[ghost](http://localhost:2368/ghost/)
Uncaught Error: Could not find module `moment-timezone` imported from `ghost-admin/app`
    u http://localhost:2368/ghost/assets/vendor-3631184082d609038638c1e169a002e7.js:13
    u http://localhost:2368/ghost/assets/vendor-3631184082d609038638c1e169a002e7.js:13
    findDeps http://localhost:2368/ghost/assets/vendor-3631184082d609038638c1e169a002e7.js:25
    u http://localhost:2368/ghost/assets/vendor-3631184082d609038638c1e169a002e7.js:13
    requireModule http://localhost:2368/ghost/assets/vendor-3631184082d609038638c1e169a002e7.js:7
    <anonymous> http://localhost:2368/ghost/assets/ghost-4416ac3815c54eac76c2590d1154a628.js:6579
[vendor-3631184082d609038638c1e169a002e7.js:13:31](http://localhost:2368/ghost/assets/vendor-3631184082d609038638c1e169a002e7.js)
```

**Comment by gbrl-almeida** [comment]
I have the same error when trying to install ghost 5.61.0 on local windows.
My admin page was stucked on loading page.
I just downgrande my local version to 5.50.4  to resolve.

**Comment by markstos** [comment]
The 5.61.0 release has been deprecated due to a broken admin area build. Downgrade to 5.60.0 or wait for 5.61.1

**Comment by daniellockyer** [comment]
This should be fixed in v5.61.1 🙂

**Comment by nebulade** [comment]
Thanks, I can confirm it fixed the issue :)
